packages/autocomplete-js/src/__tests__/autocomplete.test.ts

```diff
@@ -200,6 +201,96 @@ describe('autocomplete-js', () => {
     ).toHaveTextContent('No results template');
   });
 
+  test("doesn't render empty template on no query when openOnFocus is false", async () => {
+    const container = document.createElement('div');
+    const panelContainer = document.createElement('div');
+
+    document.body.appendChild(panelContainer);
+    autocomplete<{ label: string }>({
+      container,
+      panelContainer,
+      openOnFocus: false,
+      getSources() {
+        return [
+          {
+            getItems() {
+              return [];
+            },
+            templates: {
+              item({ item }) {
+                return item.label;
+              },
+              empty() {
+                return 'No results template';
+              },
+            },
+          },
+        ];
+      },
+    });
+
+    const input = container.querySelector<HTMLInputElement>('.aa-Input');
+
+    fireEvent.input(input, { target: { value: '' } });
+
+    await waitFor(() => {
+      expect(
+        panelContainer.querySelector<HTMLElement>('.aa-Panel')
+      ).not.toBeInTheDocument();
+    });
+
+    expect(
+      panelContainer.querySelector<HTMLElement>('.aa-Panel')
+    ).not.toBeInTheDocument();
+
+    expect(input).toHaveValue('');
+  });
+
+  test('render empty template after query when openOnFocus is false', async () => {
```

Comment:
Wdym by "after query"?

Reply:
The opposite of `no query`, maybe `on query` would be enough